qlib/workflow/expm.py

```diff
@@ -33,6 +33,10 @@ def __repr__(self):
             name=self.__class__.__name__, duri=self._default_uri, curi=self._current_uri
         )
 
+    def reset_default_uri(self, uri: Text):
+        self._default_uri = uri
```

Comment:
Is any reason that you want to add this method? From my perspective, `default_uri` should be initialized and fixed when users call `qlib.init`. It seems there is no need to have such method here.

Reply:
A simple case is:
after we run many different experiments (with different uri), we may want to compare their performance. To query the metrics of each of these experiments, we need to let R know their uri.